lib/LedgerBlockStorage.js

```diff
@@ -16,8 +16,9 @@ const {BedrockError} = bedrock.util;
 /**
  * The blocks API is used to perform operations on blocks associated with a
  * particular ledger.
+ * @memberof module:bedrock-ledger-storage-mongodb
  */
-module.exports = class LedgerBlockStorage {
+class LedgerBlockStorage {
```

Comment:
We are using this preferred syntax for exporting classes elsewhere: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/blob/master/lib/agents/consensus-agent.js#L19
```
module.exports = class Foo {
```

Reply:
Right. That breaks JSDoc.

Reply:
That's strike 1 for JSDoc.  I will approve.